_data/domains.yml

```diff
@@ -201,6 +201,12 @@ websites:
       tfa: No
       twitter: HeartInternet
 
+    - name: Hexonet
+      url: http://www.hexonet.net
+      tfa: Yes
+      software: Yes
+      doc: https://cp.hexonet.net/cp2/downloads/2FA.pdf
```

Comment:
This is the only documentation I have found. The link to 2FA instructions is located under the authentication form. 
![2016-10-17_12-17-48](https://cloud.githubusercontent.com/assets/1226001/19445949/7d2f4e42-9464-11e6-9ec4-423ed252c699.png)

There is nothing about it at https://wiki.hexonet.net/wiki/Main_Page.